build.sh

```diff
@@ -10,27 +10,6 @@ TOOLS_DIR=$SCRIPT_DIR/tools
 NUGET_EXE=$TOOLS_DIR/nuget.exe
 CAKE_EXE=$TOOLS_DIR/Cake/Cake.exe
 
-# Define default arguments.
-SCRIPT="build.cake"
```

Comment:
We use different defaults when building Cake. We use `verbose` verbosity for example, while the regular verbosity is `normal`.